.github/workflows/build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           builder: ${{ steps.buildx.outputs.name }}
           context: .
           file: ./artifacts/images/agent-build.Dockerfile
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
```

Comment:
Should not it be just arm?

Reply:
nope, it really is `linux/arm/v7`, https://docs.docker.com/desktop/multi-arch/

I _guess_ it's to make it 100% explicit if you're targeting v7 or v6